src/launch.rs

```diff
@@ -1,5 +1,12 @@
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
hmm, this should be `src/launch/mod.rs` to be consistent with the rest of the crate